dev-portal/scripts/deploy-stack.js

```diff
@@ -0,0 +1,33 @@
+const { execute, r } = require('./utils.js')
+
+const deployerConfig = require('../deployer.config.js')
+const writeConfig = require('./write-config.js')
+
+const samTemplate = deployerConfig.samTemplate || r(`../../cloudformation/template.yaml`)
+const packageConfig = deployerConfig.packageConfig || r(`../../cloudformation/packaged.yaml`)
+const stackName = deployerConfig.stackName
+const buildAssetsBucket = deployerConfig.buildAssetsBucket
+const siteAssetsBucket = deployerConfig.siteAssetsBucket
+const apiAssetsBucket = deployerConfig.apiAssetsBucket
+
+function main() {
+  Promise.resolve()
+    .then(() => execute(`sam package --template-file ${samTemplate} --output-template-file ${packageConfig} --s3-bucket ${buildAssetsBucket}`, true))
```

Comment:
10/10 on swallowing the misleading "use cfn deploy with these wrong args!" message.